package-lock.json

```diff
@@ -1509,7 +1509,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
I've never really understood what causes this change from https to http. I think it has stopped happening to me, however, and I'm running npm v6.8.0. So if you're not on 6.8.0 or 6.9.0, maybe try upgrading and running 'npm install' again to see if that prevents these changes?

It would be nice to avoid changing this file unnecessarily, but I don't think it is worth a lot of your time to do that. Actually, since it doesn't look like you've changed package.json, I suppose you could just `git checkout package-lock.json` to restore the version from master.